benchmarks/tasks.py

```diff
@@ -325,28 +325,27 @@ def sync_external_repos(self):
 @celery_app.task(bind=True)
 def daily_benchmark_progress(self):
     now = timezone.now()
-    interval = relativedelta(days=1)
+    yesterday = now - relativedelta(days=1)
 
-    results = models.Result.objects.compare_progress(now, interval)
```

Comment:
do we still need compare_progress function? It's left there in the ResultManager and the only place it's used is tests.